styles.css

```
@@ -0,0 +1,132 @@
+body {
+    font-family: 'Lucida Sans Unicode', Arial, sans-serif;
+    margin: 0;
+    padding: 0;
+    background: linear-gradient(to right, #f4f4f4, #e0e0e0);
+    line-height: 2.0;
+    position: relative;
+}
+
+h1 {
+    color: #333;
+}
+
+section {
+    margin: 20px;
+    padding: 20px;
+    background-color: #fff;
+    border: 1px solid #ddd;
+    border-radius: 5px;
+}
+
+section > h2 {
+    margin-bottom: 10px;
+}
+
+section > p,
+section > ol,
+section > ul {
+    margin-top: 0;
+    margin-bottom: 10px;
+}
+
+section > div {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    padding: 20px;
+}
+
+video {
+    width: 50%;
+    height: auto;
+    border-radius: 5px;
+    padding: 20px;
+    margin: 2.5%;
+    flex: 1;
+    }
+    
+    video:hover {
+        transform: scale(1.05);
+        transition: transform 0.3s ease;
+    }
```

```suggestion
video {
    width: 50%;
    height: auto;
    border-radius: 5px;
    padding: 20px;
    margin: 2.5%;
    flex: 1;
    
    &:hover {
        transform: scale(1.05);
        transition: transform 0.3s ease;
    }
}
   
```

Puede usar CSS Nesting